test_communication/test/test_publisher_subscriber_python.py.in

```diff
@@ -0,0 +1,38 @@
+import sys
+from launch import LaunchDescriptor
+from launch.exit_handler import primary_exit_handler, default_exit_handler
+from launch.launcher import DefaultLauncher
+
+
+def test_publisher_subscriber():
+    ld = LaunchDescriptor()
+
+    ld.add_process(
+        cmd=[sys.executable,
+             '@TEST_PUBLISHER_PY_EXECUTABLE@',
+             '@TEST_MESSAGE_TYPE@',
+             '-r', '@TEST_RMW_IMPLEMENTATION_PY@'],
```

Comment:
Indeed. Python tests do need to get the rmw_implementation_identifier as a parameter to select it through `rclpy` functions, while cpp tests can just link against the right one.
An alternative could be to set the RCLPY_IMPLEMENTATION env variable before each test and just use that in the python tests. This way we could use the same template.

If we extend `multi_target` this way, we will want to also test pub/sub using the same rmw_implementation. If we do this we could also get rid of `custom_test` and do everything in `multi_target` (which may need another name at this point).